lib/gitlab/ci/templates/pipeline/AndroidTemplate.yml

```diff
@@ -13,12 +13,16 @@ variables:
   DEFAULT_IMAGE: "jangrewe/gitlab-ci-android"
   DEPLOY_DEBUG_APK_SLACK_MESSAGE: "Hello Team! Here is the latest debug APK from branch `${CI_COMMIT_REF_NAME}`. It was triggered by: `${CI_PIPELINE_SOURCE}`."
```

Comment:
Nit, maybe change to the below:

DEPLOY_DEBUG_SLACK_MESSAGE: "Hello Team! Here is the latest debug APK/AAR from branch `${CI_COMMIT_REF_NAME}`. It was triggered by: `${CI_PIPELINE_SOURCE}`."